charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -280,6 +289,11 @@ spec:
     custom:
       PULSAR_PREFIX_additionalServlets: "license-plugin-servlet"
       PULSAR_PREFIX_additionalServletDirectory: "./brokerAdditionalServlet"
+      {{- if .Values.tls.broker.enabled }}
+      PULSAR_PREFIX_tlsCertificateFilePath: "/pulsar/certs/broker/tls.crt"
+      PULSAR_PREFIX_tlsKeyFilePath: "/pulsar/certs/broker/tls.key"
+      PULSAR_PREFIX_tlsTrustCertsFilePath: "/pulsar/certs/ca/ca.crt"
```

Comment:
wouldn't this conflict with the `tlsTrustCertsFilePath` defined below?

Reply:
Clarify the `tlsTrustCertsFilePath` priority in new commit.